.github/workflows/nightly.yml

```diff
@@ -45,3 +51,59 @@ jobs:
 
     - name: Run tests under valgrind
       run: ${{github.workspace}}/test/test_valgrind.sh ${{github.workspace}} ${{github.workspace}}/build ${{matrix.tool}}
+
+  coverity:
+    name: Coverity
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Install apt packages
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y cmake hwloc libhwloc-dev libjemalloc-dev libnuma-dev libtbb-dev
+
+    - name: Download Coverity
+      run: |
+        wget -nv https://scan.coverity.com/download/linux64 --post-data "token=${{ secrets.COVERITY_SCAN_TOKEN }}&project=oneapi-src%2Funified-memory-framework" -O coverity_tool.tgz
+
+    - name: Extract Coverity
+      run: |
+        tar xzf coverity_tool.tgz
```

Comment:
I think by default it will untar the files into the current dir, so perhaps, not to mix with our repo files, it could be unpacked into a sub-dir...?

Reply:
It works in the way it should.